bricksrc/sensor.py

```diff
@@ -577,6 +595,21 @@
                             },
                         },
                     },
+                    "Energy_Usage_Sensor": {
+                        "parents": [BRICK.Energy_Sensor],
+                        "tags": [TAG.Point, TAG.Sensor, TAG.Energy, TAG.Usage],
+                        "subclasses": {
+                            "Yearly_Usage_Energy_Sensor": {
```

Comment:
The order of tags in the class names change in these subclasses from `Energy_Usage_Sensor` to `Usage_Energy_Sensor`. I prefer the former one though I have no strong opinion.

Reply:
Sorry; I'm very confused by what you mean there. Can you rephrase? Are you saying the order of words in `"tags"` is inconsistent? Or the class names are inconsistent?

Reply:
I meant the class names. I want to change `Yearly_Usage_Energy_Sensor` to `Yearly_Energy_Usage_Sensor` (same for its siblings). Their superclass's name is `Energy_Usage_Sensor`.

Reply:
Ah, makes sense. Good change